src/v/pandaproxy/schema_registry/storage.h

```diff
@@ -1071,21 +1044,51 @@ struct consume_to_store {
                 co_await _store.set_compatibility(val->compat);
             }
         } catch (const exception& e) {
-            vlog(plog.debug, "Ignoring: {}: {}", key, e.what());
+            vlog(plog.debug, "Error replaying: {}: {}", key, e.what());
         }
     }
 
-    ss::future<> apply(delete_subject_key key, delete_subject_value val) {
+    ss::future<> apply(
+      model::offset offset,
+      delete_subject_key key,
+      std::optional<delete_subject_value> val) {
+        // Out-of-place events happen when two writers collide.  First
+        // writer wins: disregard subsequent events whose seq field
+        // doesn't match their actually offset.
+        if (val.has_value() && offset != key.seq) {
```

Comment:
`node_id`?

Reply:
We only put node ID in there to ensure that compaction doesn't collapse two same-sequence keys from different nodes.  It's unnecessary to look at the node ID during replay -- an event's sequence matching its position in the topic is sufficient